doc/readme.md

```diff
@@ -0,0 +1,335 @@
+
+Animation können wir mit dem `AnimationBuilder` erzeugen, indem wir `AnimationBuilder.buildAnimation` aufrufen und als Parameter das Verzeichnis mit den Bildern übergeben, die zu einer Animation zusammengefasst werden sollen. Das `AnimationComponent` wird im Konstruktor erstellt.
+
+``` java
+package ecs.entities;
+
+import dslToGame.AnimationBuilder;
+import ecs.components.AnimationComponent;
+import ecs.components.PositionComponent;
+import graphic.Animation;
+import tools.Point;
+
+public class MyHero extends Entity {
+
+    public MyHero() {
+        super();
+        new PositionComponent(this, new Point(0, 0));
+        Animation idleLeft = AnimationBuilder.buildAnimation("character/knight/idleLeft");
+        Animation idleRight = AnimationBuilder.buildAnimation("character/knight/idleRight");
+        new AnimationComponent(this, idleLeft, idleRight);
+    }
+}
+
+```
+Wenn Sie das Spiel jetzt starten, sollten SIe ihren Helden sehen:
+
+![Animation](figs/animation.gif)
+
+### Held bewegen
+
+Damit der Held vom `VelocitySystem` bewegt werden kann, benötigt er das `VelocityComponent`. Das `VelocityComponent` speichert die Maximalgeschwindigkeit des Helden, sowie die aktuelle geschwindigkeit des Helden.
+
+Damit wir den Helden auch per Tastatur steuern können, benötigt er das `PlayableComponent`, denn dann wird das `PlayerSystem` die Werte des Helden auf Knopfdruck aktualisieren.
+
+*Hinweis: Mehr zur Konfiguration der Tastenbelegung erfahren Sie [hier](configuration/readme.md).*
+
+Genau wie die anderen `Component`s fügen wir das `VelocityComponent` und das `PlaybaleComponent` zum Konstruktor hinzu.
+
+``` java
+package ecs.entities;
+
+import dslToGame.AnimationBuilder;
+import ecs.components.AnimationComponent;
+import ecs.components.PlayableComponent;
+import ecs.components.PositionComponent;
+import ecs.components.VelocityComponent;
+import graphic.Animation;
+import tools.Point;
+
+public class MyHero extends Entity {
+
+    public MyHero() {
+        super();
+        new PositionComponent(this, new Point(0, 0));
+        Animation idleLeft = AnimationBuilder.buildAnimation("character/knight/idleLeft");
+        Animation idleRight = AnimationBuilder.buildAnimation("character/knight/idleRight");
+        new AnimationComponent(this, idleLeft, idleRight);
+        new PlayableComponent(this);
+        float xSpeed = 0.3f;
+        float ySpeed = 0.3f;
+        Animation runLeft = AnimationBuilder.buildAnimation("character/knight/runLeft");
+        Animation runRight = AnimationBuilder.buildAnimation("character/knight/runRight");
+        new VelocityComponent(this, xSpeed, ySpeed, runLeft, runRight);
+    }
+}
+
+```
+
+Jetzt sollten Sie ihren Helden bewegen können:
+
+![Bewegen](figs/move.gif)
+
+### Nächstes Level laden
```

Comment:
Das spielt für den ersten eigenen Helden eigentlich keine Rolle - sonst müsste man ja auch noch weiter gehen und andere Dinge erklären/analysieren? Für mich gehört das technisch zur "Code-Analyse". Ich würde gern die Einleitung und den Überblick, das How-To und die Code-Analyse gliederungstechnisch sauber trennen. Bitte mit passenden Überschriften arbeiten.